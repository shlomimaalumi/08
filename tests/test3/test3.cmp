|RAM[5000|RAM[5001|RAM[5002|
|     21 |     44 |      4 |
