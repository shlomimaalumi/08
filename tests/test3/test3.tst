load test3.asm,
output-file test3.out,
compare-to test3.cmp,
output-list RAM[5000]%D1.6.1 RAM[5001]%D1.6.1 RAM[5002]%D1.6.1;

repeat 1000000 {
  ticktock;
}

output;
