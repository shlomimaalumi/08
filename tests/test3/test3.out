|RAM[5000|RAM[5001|RAM[5002|
